src/clients/c++/perf_client/perf_utils.cc

```diff
@@ -166,3 +172,68 @@ ScheduleDistribution<Distribution::CONSTANT>(const double request_rate)
           std::chrono::duration<double>(1.0 / request_rate));
   return [period](std::mt19937& /*gen*/) { return period; };
 }
+
+nic::Error
+CreateSharedMemoryRegion(std::string shm_key, size_t byte_size, int* shm_fd)
```

Comment:
Use these fucntions from shm utils instead of redefining here. Instead move the CUDA shared memory functions here if needed

Reply:
Done.